internal/cmd/flags.go

```diff
@@ -63,6 +64,10 @@ func (f *flags) bindCallTimeout(flagSet *pflag.FlagSet) {
 	flagSet.StringVar(&f.callTimeout, "call-timeout", "60s", "The maximum time to for all calls to be completed.")
 }
 
+func (f *flags) bindConfigData(flagSet *pflag.FlagSet) {
+	flagSet.StringVar(&f.configData, "config-data", "", "The configuration data to use instead of reading prototool.yaml or prototool.json files.\nThis will act as if there is a configuration file with the given data in the current directory, and no other configuration files recursively.\nThis is an advanced feature and is not recommended to be generally used.")
```

Comment:
I think we should shorten this description. The first sentence will likely suffice:
```
The configuration data to use instead of reading prototool.yaml or prototool.json files.
```

Reply:
Hmmm, I disagree, let's cover this today offline?